### PR TITLE
pandas detection and event tables / summary tables / Excel export

### PR DESCRIPTION
**Pull request by sommerc**
Dear Fabrice,

I wrote some (mostly) pandas based functions to extract features from mouse detection and mouse events. 

Detection:
 * distance traveled
 * time spent
 * avg speed 

Events:
 * number of events
 * duration statistics

This can be easily exported to Excel (what most biologists want :)

I added a new example notebook to the example folder on how to use it. 

On the way, I did some tiny restructuring to the original code base.

Perhaps, you think this is useful for more people and have some hints on how to improve the code.

Thanks and cheers to Paris,
Chris

### CODE REVIEW

File: LMT/lmtanalysis/Measure.py [hunk 1]
```diff
@@ -58,6 +58,12 @@
 ''' Cage center in 50x50cm area'''
 cageCenterCoordinates50x50Area = Point( 256, 208 )
 
+''' Size of arena in cm '''
+ARENA_SIZE = 50
+
+''' Margin to define center region in cm (chosen to have same area as non-center)'''
+CENTER_MARGIN = 7.32
```
Comment (fdechaumont) [inline LMT/lmtanalysis/Measure.py:65]: Is that the area between the floor and the top of the wall ?

Comment (sommerc) [reply LMT/lmtanalysis/Measure.py:65]: that's only a margin to define an inner central area of the arena.
so the 50x50 has an inner square of size (50-2*7.32)^2

**Comment by fdechaumont** [comment]
Thanks very much for your interest, and for your suggestions. Very well written !

Best,

Fabrice

**Comment by sommerc** [comment]
Thanks!

**Pull request merged**